src/Ingredients/DisplayRecipes.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+
+namespace BreadAndIfit\Ingredients;
+
+
+class DisplayRecipes
+{
+    /**
+     *takes the JSON data returned from the API request and returns a string with HTML markup
+     *
+     * @param $json API response in JSON format
+     * @return string relevant recipes based on user selected ingredients or a error message string
+     */
+    public static function outputRecipes($json)
+    {
+        $recipes = json_decode($json)->results;
+
+        $validate = self::validateRecipe($recipes);
+
+        if ($validate) {
+            return self::outputHTML($recipes);
+        } else {
+            return '<div id="mainPannel">
+                    <div class="row recipe mx-auto">
+                        <div class="col-10 col-lg-8">
+                            <h5 class="card-title">Invalid response received from puppy API</h5>                   
+                        </div>
+                    </div>
+                </div>';
+        }
+    }
+
+    /**
+     *Checks that the JSON_decode function has returned a valid decoded array of recipe data
+     *
+     * @param $recipes an array that is returned after running through the JSON_decode function which should contain an array of recipe data
+     * @return bool returns boolean to show whether the response data from the API is in a valid format
+     */
+    private static function validateRecipe($recipes): bool
+    {
+        if (!is_array($recipes)) {
```

Comment:
this can be shortened to:
` return is_array($recipes);`

Reply:
changed and sorted